docs/API/NuGet-Protocols.md

```diff
@@ -13,77 +13,89 @@ ms.assetid: ba1d9742-9f1c-42ff-8c30-8e953e23c501
 
 # optional metadata
 
-description: The evolving nuget.org protocols to interact with nuget.org by NuGet clients.
+description: The evolving NuGet.org protocols to interact with NuGet clients.
 ms.reviewer:
 - kraigb
 - karann-msft
 
 ---
-# NuGet Protocols
+# NuGet.org Protocols
 
-To interact with nuget.org, clients need to follow certain protocols. Because these protocols keep evolving, clients must identify the protocol version they use when calling nuget.org APIs. This allows nuget.org to introduce changes in a non-breaking way for the old clients. 
+To interact with NuGet.org, clients need to follow certain protocols. Because these protocols keep evolving, clients
+must identify the protocol version they use when calling specific nuget.org APIs. This allows nuget.org to introduce
+changes in a non-breaking way for the old clients.
+
+These APIs are specific to NuGet.org and there is no expectation for other NuGet server implementations to introduce
```

Comment:
What does `These` refer to here?
Maybe make the title "NuGet.org-only protocols" or something like that? I read over that and was thinking "why wouldn't I implement them?" - Only after reading the verify-keys API I realized "oh, right - NuGet.org only".